dragonfly/block/break_info.go

```diff
@@ -89,15 +94,18 @@ var axeEffective = func(t tool.Tool) bool {
 	return t.ToolType() == tool.TypeAxe
 }
 
+// shearsEffective is a convenience function for blocks that are effectively mined with shears.
+var shearsEffective = func(t tool.Tool) bool {
+	return t.ToolType() == tool.TypeShears
+}
+
 // shovelEffective is a convenience function for blocks that are effectively mined with an axe.
 var shovelEffective = func(t tool.Tool) bool {
 	return t.ToolType() == tool.TypeShovel
 }
 
 // alwaysHarvestable is a convenience function for blocks that are harvestable using any item.
-var alwaysHarvestable = func(t tool.Tool) bool {
-	return true
-}
+var alwaysHarvestable = alwaysEffective
```

Comment:
Once you change `alwaysEffective` to `neverEffective` (see comment below), don't forget to change this back.